scripts/install_plugins.sh

```diff
@@ -5,32 +5,84 @@ CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source "$CURRENT_DIR/shared_functions.sh"
 
 clone() {
-	local plugin=$1
-	cd $SHARED_TPM_PATH &&
+	local plugin; plugin="$1"
```

Comment:
To avoid losing the exit status in a subshell, it's not applicable here but I do it as thumb rule, actually I use to avoid completely `local`.., but since `local` is used everywhere in tpm I try to keep the changes to the minimal.

https://github.com/koalaman/shellcheck/wiki/SC2155